Jenkinsfile

```diff
@@ -2,38 +2,194 @@
 
 node('us-east-1 && ubuntu && docker && !gpu') {
 
-    def hash = ""
-    
-    stage('Checkout') {
-        checkout scm
-        def delim = '"'
-        hash = sh(returnStdout: true, script: """curl https://api.github.com/repos/cliqz-oss/cliqz-android/branches/upstream-releases | grep -m 1 sha | cut -d '${delim}' -f 4""").trim()
-    }
-    def dockerTag = "${hash}"
-    def baseImageName = "browser-f/android:${dockerTag}"
-    docker.image("141047255820.dkr.ecr.us-east-1.amazonaws.com/${baseImageName}").inside {
+    def dockerTag = ""
+    def apk = ""
+    // def instance_id = sh(returnStdout: true, script: '''
```

Comment:
can those comments be removed?

Reply:
Trying them out to check if it works. Removed them.